### DISCUSSION TITLE
QA flow in pull-requests

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#5840] [General]
Greetings @prury and @degenofdotsama, welcome to @kodadot/qa-guild. You are now both enlisted to help us ensure that our ongoing issues are addressed seamlessly and alleviate tasks such as verifying pull requests from developers and designers.

At times, issue descriptions may be vague. Nonetheless, our aim is to ensure that all functions work according to their intent without compromising existing features. Anytime you notice a malfunction, feel free to inform the developers and reevaluate prior to merging.

For any bug encountered, kindly open an issue or merge with another similar bug report.

Our current workflow is as follows: create an issue -> developer analyzes and works on it -> submit a pull request that closely aligns with the issue description. Your responsibility begins when scrutinizing the pull request, previewing the deployment, and verifying its functionality.

If everything works, use the label works-for-me.
If something is broken, use the label changes-requested.

In the future, we may integrate the qa-guild into the formal review process. Presently, we will assess your performance during this trial period.

Should you have any questions or need assistance, please reach out here or contact @helloitsdamsky.

For a better understanding of the KodaDot QA process, please refer to the following document:

https://github.com/kodadot/nft-gallery/blob/main/QA.md
Additionally, depending on your expertise, contributing tests and enhancing the coverage of various functionalities are highly appreciated.

**Comment by 1Mike-AF** [comment]
thank you.

**Comment by prury** [comment]
allright, on it
